clippy_lints/src/utils/internal_lints.rs

```diff
@@ -207,3 +218,28 @@ impl<'a, 'tcx: 'a> Visitor<'tcx> for LintCollector<'a, 'tcx> {
         NestedVisitorMap::All(&self.cx.tcx.hir)
     }
 }
+
+
+
+pub struct PreferFxHash;
+
+impl LintPass for PreferFxHash {
+    fn get_lints(&self) -> LintArray {
+        lint_array!(PREFER_FX_HASH)
+    }
+}
+
+impl EarlyLintPass for PreferFxHash {
+    fn check_ident(&mut self, cx: &EarlyContext<'_>, ident: Ident) {
+        if let Some(msg) = {
+            if ident.to_string().contains("HashMap") {
```

Comment:
Won't this lint `FxHashMap` as well? Shouldn't it be `==` instead of `contains`? :thinking:

Reply:
Yeah you're right. I don't know `Ident`, so I'm looking for help on this one..

Reply:
Went the easy way..